static/js/core.js

```diff
@@ -68,5 +53,5 @@ core.mobileNav = function () {
     }
 };
 
-core.svgFallback();
+core.supportsSvg();
```

Comment:
Why is this necessary it will just return a boolean?

Reply:
Only in modern browsers, it works okay in older browsers - https://stackoverflow.com/questions/43317691/why-document-implementation-hasfeature-always-return-true

Reply:
I understand it will only return on modern browsers but it will just return `true` which does nothing else? Is this function used by anything?